src/httpServer/server.ts

```diff
@@ -93,7 +93,12 @@ export class OperonHttpServer {
           try {
             // Check for auth first
             if (defaults?.authMiddleware) {
-              const res = await defaults.authMiddleware({name: ro.name, requiredRole: ro.getRequiredRoles(), koaContext: koaCtxt});
+              const res = await defaults.authMiddleware({
+                name: ro.name, requiredRole: ro.getRequiredRoles(), koaContext: koaCtxt,
+                logger: oc.logger, span: oc.span,
+                getConfig: (key:string, def)=>{return oc.getConfig(key, def);},
+                query: (qry, ...args) => {return operon.userDatabase.queryFunction(qry, ...args);}
```

Comment:
What about `query` instead of `qry` 😂 ? (Yes I know, ironic coming from the person advocating the use of `ctx`.)